pages/common/mitmproxy.md

```diff
@@ -0,0 +1,11 @@
+# mitmproxy
+
+> An interactive man-in-the-middle HTTP proxy.
```

Comment:
It's a good description but I think proxies are always man-in-the-middle. That's being annoying because you're literally just spelling out the name. I think they threw in the MITM part because it's used especially for MITM attacks? Or, well, mitm analysis.

Please correct me if I'm wrong.